cmd/lora-gateway-bridge/cmd/configfile.go

```diff
@@ -117,6 +117,26 @@ clean_session={{ .Backend.MQTT.CleanSession }}
 # a random id will be generated. This requires clean_session=true.
 client_id="{{ .Backend.MQTT.ClientID }}"
 
+# Last Will topic
+#
+# Set Last Will topic, where Last Will message will be published
+last_will_topic="{{ .Backend.MQTT.LastWillTopic }}"
+
+# Last Will payload
+#
+# Set Last Will payload to be publish on disconnect
+last_will_payload="{{ .Backend.MQTT.LastWillPayload }}"
+
+# Last Will QoS
+#
+# Set Last Will payload QoS
+last_will_qos="{{ .Backend.MQTT.LastWillQoS }}"
+
+# Last Will retain
+#
+# Set Last Will retain option
+last_will_retain="{{ .Backend.MQTT.LastWillRetain }}"
```

Comment:
I'm not sure if this should be retained. You only want to send it on a non-graceful disconnect. Making it retained means that when the gateway re-connects successful, each MQTT client subscribing would directly receiving the last-will, while the gateway is still connected.

I think it is better to remove this and always use `false`.

Reply:
I suggest leaving retain flag, even if it's false by default. In my case I'm using last will message like this: OnConnect publish {"ONLINE":1}, last will message is {"ONLINE":0}.

When these messages are retained, I can easily find out if this device is ONLINE or not.

Do you mind leaving this option and by default make it false?

Reply:
Thinking about this again, there is no reason to keep retain because we removed the payload. 

So I suggest, if you don't mind, leave payload as an option (by default empty string in order to be replaced by the mac, with retain false) and if there is a payload replace the mac with the payload and use the retain flag too. 

What do you think?